src/pilot/assign_role.py

```diff
@@ -478,10 +485,10 @@ def define_storage_logical_disks(drac_client, raid_controllers):
     return logical_disks
 
 
-def define_storage_operating_system_logical_disk(physical_disks, drac_client,
+def define_operating_system_logical_disk(physical_disks, drac_client,
                                                  raid_controller_name):
     (os_logical_disk_size_gb,
-     os_physical_disk_names) = find_physical_disks_for_storage_os(
+     os_physical_disk_names) = find_physical_disks_for_os(
         physical_disks)
```

Comment:
can try to wrap line 491 and 492 into one if it not as long anymore

Reply:
Done